src/mappings/oba-efo.sssom.tsv

```diff
@@ -146,4 +146,30 @@ GO:1903492	response to aspirin	response to stimulus	skos:relatedMatch	OBA:204001
+EFO:0010948	lower face morphology measurement	measurement	skos:broadMatch	OBA:VT0003743 	 facial morphology trait		HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0004300	longevity	measurement	skos:exactMatch	OBA:VT0005372 	 life span trait		HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0004505	telomere length	measurement	skos:exactMatch	OBA:2050100			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0007842	chin morphology measurement	measurement	skos:exactMatch	OBA:2050101			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0009286	heart relative wall thickness	measurement	skos:broadMatch	OBA:2050102			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0004844	hip bone size	measurement	skos:exactMatch	OBA:2050103			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0600005	ganglion cell inner plexiform layer thickness measurement	measurement	skos:broadMatch	OBA:2050104			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0009278	cranial vault morphology measurement	measurement	skos:exactMatch	OBA:2050105			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0007844	forehead morphology measurement	measurement	skos:exactMatch	OBA:2050106			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0005053	internal carotid intimal medial thickness	measurement	skos:broadMatch	OBA:2050107			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0004860	common carotid intimal medial thickness	measurement	skos:broadMatch	OBA:2050108			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0007739	left superior temporal gyrus thickness measurement	measurement	skos:broadMatch	OBA:2050109			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0600002	retinal layer thickness measurement	measurement	skos:exactMatch	OBA:2050110			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0600004	retinal nerve fibre layer thickness measurement	measurement	skos:exactMatch	OBA:2050111			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0007823	facial hair thickness measurement	measurement	skos:exactMatch	OBA:2050112			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0008593	ldl particle size measurement	measurement	skos:exactMatch	OBA:2050113			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0008594	vldl particle size measurement	measurement	skos:exactMatch	OBA:2050114			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0008592	hdl particle size measurement	measurement	skos:exactMatch	OBA:2050115			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744
+EFO:0007117	carotid artery intima media thickness	measurement	skos:broadMatch	OBA:2050108			HumanCurated	EFO	OBA	1	2023-02-09				orcid:0000-0001-6757-4744

```

Comment:
Please, remove this`src/mappings/oba-efo.sssom.tsv` version from the PR, because this version of the table has fewer lines than [the OBA Ontology Development (GWAS, EFO, VT) gsheet]( https://docs.google.com/spreadsheets/d/13qh7dLE38vMyz91oRqj6GzKjFohNazNKAJxKG6Plw1o/edit#gid=1005741851).
Did you follow the 'Updating SSSOM mapping files' section of the [Local development workflows with OBA](https://obophenotype.github.io/bio-attribute-ontology/editors-guide/)?
This tsv should agree with the [OBA Ontology Development (GWAS, EFO, VT) gsheet]( https://docs.google.com/spreadsheets/d/13qh7dLE38vMyz91oRqj6GzKjFohNazNKAJxKG6Plw1o/edit#gid=1005741851), but it does not.